### PR TITLE
Add Awesome Python Decorator list

### PR DESCRIPTION
**Pull request by lord63**
This list collects some resource for learning python decorator. It lists the built-in python decorator, what more, also shows some real projects with a short description about how they use python decorator.

The repo site is: https://github.com/lord63/awesome-python-decorator

### CODE REVIEW

**Comment by aleksandar-todorovic** [comment]
I like this! The only suggestion I have is adding a short sentence or two about what decorators are before suggesting people to go and visit those links that explain the topic in full detail.

**Comment by lord63** [comment]
@aleksandar-todorovic Glad you like it! :laughing: 

You suggestion makes sense, for example we can add a line:

> A decorator is a function or a class that wraps a function or a method.

But in my opinion, python decorator is an advanced topic in python, and I prefer listing the PEPs so you can get a better feeling about it  :)

**Comment by sindresorhus** [comment]
> But in my opinion, python decorator is an advanced topic in python, and I prefer listing the PEPs so you can get a better feeling about it

I think you're missing the point. You should clearly state what the list is about otherwise visitors will go there, not know what it's about, and leave. Might be easier for you if you think of an awesome list as a product landing page. Your job is to retain users.

**Comment by sindresorhus** [comment]
I would also add some blog posts and videos about decorators.

File: readme.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Education](https://github.com/hsavit1/Awesome-Swift-Education)
 	- [Playgrounds](https://github.com/uraimo/Awesome-Swift-Playgrounds)
 - [Python](https://github.com/vinta/awesome-python)
+	- [Python Decorator](https://github.com/lord63/awesome-python-decorator)
```
Comment (davisonio) [inline readme.md]: If this is under the 'Python' list then the link can simple be called 'Decorator' or 'Decorators' even.

**Comment by sindresorhus** [comment]
Closing for lack of activity. Happy to reconsider in a month when you've resolved the feedback and worked on the list some more.

**Pull request closed without merging**